src/pages/PersonalisationWizardPage.js

```diff
@@ -21,6 +21,15 @@ class PersonalisationWizardPage extends BasePersonalisationPage {
         var subpage = this.state.subpage - 1;
 
         if (subpage < 0) {
+            /*
+             * FIXME: If we use a real URL for each subpage,
```

Comment:
@danni I'd love to hear your thoughts on this approach to getting the back/forwards to work nicely in personalisation.

I think it'll let us use regular history while still preserving the desired behavior of having the `back` button skip over the personalisation questions.